test/e2e/storage/drivers/in_tree.go

```diff
@@ -91,6 +91,10 @@ func InitNFSDriver() testsuites.TestDriver {
 			Name:             "nfs",
 			InTreePluginName: "kubernetes.io/nfs",
 			MaxFileSize:      testpatterns.FileSizeLarge,
+			SupportedSizeRange: volume.SizeRange{
+				Min: "1Gi",
+				Max: "10Gi",
```

Comment:
Why 10Gi? The same applies to all other drivers.

If there is no known limit, then it would be better to leave it unset.

Also note that 1Gi might be too small for some drivers. Better use 5Gi, that is known to work.

Reply:
If Min or Max is unset in those drivers, it will fail to calculate intersection and skip the test because
 ```	go
func getSizeRangesIntersection(first volume.SizeRange, second volume.SizeRange) string {
	var firstMin, firstMax, secondMin, secondMax resource.Quantity
	var err error
	if firstMin, err = resource.ParseQuantity(first.Min); err != nil {
		framework.Skipf("Failed parsing quantity %s", first.Min)
	}

	if firstMax, err = resource.ParseQuantity(first.Max); err != nil {
		framework.Skipf("Failed parsing quantity %s", first.Max)
	}
...
```

Reply:
Correct.

Reply:
`getSizeRangesIntersection` could (or should?) be changed to allow empty strings and treat them as "no minimum or maximum set". I prefer that approach over artificially setting limits for a driver that are not based on anything in the driver's implementation. Same for test suites.

This will lead to additional corner cases, but those can be handled. For example, if both ranges are unset, we can fall back to some default size.

Reply:
ok, if sizeRange is not set , use default value